app/views/layouts/application.html.erb

```diff
@@ -42,9 +42,13 @@
           <button type="button" role="button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation" data-module="govuk-button"><%= t 'layout.application.menu' %></button>
           <nav>
             <ul id="navigation" class="govuk-header__navigation " aria-label="Top Level Navigation">
+              <li class="govuk-header__navigation-item">
```

Comment:
this should also be constrained by `<% if user_signed_in? %>` and we'll add the role later